access/handler.go

```diff
@@ -405,6 +408,28 @@ func (h *Handler) GetEventsForBlockIDs(
 	}, nil
 }
 
+// GetLatestProtocolStateSnapshot returns the latest serializable Snapshot
+func (h *Handler) GetLatestProtocolStateSnapshot(ctx context.Context, req *access.GetLatestProtocolStateSnapshotRequest) (*access.ProtocolStateSnapshotResponse, error) {
+	snapshot, err := h.api.GetLatestProtocolStateSnapshot(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	serializable, err := inmem.FromSnapshot(snapshot)
+	if err != nil {
+		return nil, err
+	}
+
+	data, err := json.Marshal(serializable)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
I would suggest we put all of this in a new method in https://github.com/onflow/flow-go/blob/master/engine/common/rpc/convert/convert.go and then call that method. That would be inline with all other  calls.

Reply:
`Convert.SnapshotToBytes(snapshot protocol.Snapshot)  ([]byte, error)`